pkg/vcs/linux.go

```diff
@@ -214,37 +214,64 @@ func (ctx *linux) Bisect(bad, good string, trace io.Writer, pred func() (BisectR
 }
 
 func (ctx *linux) addMaintainers(com *Commit) {
-	if len(com.CC) > 2 {
+	if len(com.Recipients) > 2 {
 		return
 	}
-	list := ctx.getMaintainers(com.Hash, false)
-	if len(list) < 3 {
-		list = ctx.getMaintainers(com.Hash, true)
+	mtrs := ctx.getMaintainers(com.Hash, false)
+	if len(mtrs) < 3 {
+		mtrs = ctx.getMaintainers(com.Hash, true)
 	}
-	com.CC = email.MergeEmailLists(com.CC, list)
+	com.Recipients = append(com.Recipients, mtrs...)
+	sort.Sort(com.Recipients)
 }
 
-func (ctx *linux) getMaintainers(hash string, blame bool) []string {
+func (ctx *linux) getMaintainers(hash string, blame bool) Recipients {
 	// See #1441 re --git-min-percent.
 	args := "git show " + hash + " | " +
 		filepath.FromSlash("scripts/get_maintainer.pl") +
-		" --no-n --no-rolestats --git-min-percent=20"
+		" --git-min-percent=20"
 	if blame {
 		args += " --git-blame"
 	}
 	output, err := osutil.RunCmd(time.Minute, ctx.git.dir, "bash", "-c", args)
 	if err != nil {
 		return nil
 	}
-	var list []string
-	for _, line := range strings.Split(string(output), "\n") {
-		addr, err := mail.ParseAddress(line)
+	return ParseMaintainersLinux(output)
+}
+
+func ParseMaintainersLinux(text []byte) Recipients {
+	lines := strings.Split(string(text), "\n")
+	if len(lines) == 1 && len(lines[0]) < 1 {
```

Comment:
It will be ignored by the code below but it will return a slice containing LKML, I thought the correct way was to return an empty slice if the input is an empty string.